community/Wings Fail2ban Setup.md

```diff
@@ -0,0 +1,66 @@
+# Fail2ban wings configuration
+
+Fail2ban is a program that works with iptables to better secure your server and services from brute force login attempts
+
+### Step 1: if you havent already Using your preferred package manager
+(to my best knowledge, iptables will come with fail2ban when you install it, but if not install that too)
+
+`apt-get install fail2ban`
+
+### Step 2: After you install Fail2Ban Copy some files, If you already have local files, skip these steps
```

Comment:
Headers are normally short and concise with additional information specified in a paragraph below.